src/libraries/System.Text.Json/gen/JsonSourceGenerator.Roslyn3.11.cs

```diff
@@ -27,7 +28,7 @@ public sealed partial class JsonSourceGenerator : ISourceGenerator
         /// <param name="context"></param>
         public void Initialize(GeneratorInitializationContext context)
         {
-            context.RegisterForSyntaxNotifications(() => new SyntaxContextReceiver());
+            context.RegisterForSyntaxNotifications(() => new SyntaxContextReceiver(context.CancellationToken));
```

Comment:
```suggestion
            context.RegisterForSyntaxNotifications(() => new SyntaxContextReceiver(CancellationToken.None));
```

Reply:
I like that the vibe remains cancellable, but this code is now explicit in that is not properly passing a CT along. :-)

Reply:
Would it make sense to include the rationale for not flowing the token in a comment for posterity?

Reply:
Definitely. I'll add that.